src/reactor/reactor.spec.ts

```diff
@@ -541,6 +541,175 @@ describe('reactor/reactor with cycles', () => {
+            });
+        });
+
+        context('when an error occurs in any reactor', () => {
+            let latestValue: string;
+            beforeEach('start an unstable reactor', () => {
+                d$.react(v => {
+                    latestValue = v;
+                    if (v === 'error in reactor') {
+                        throw new Error(v);
+                    }
+                });
+            });
+
+            it('should throw on Atom#set', () => {
+                expect(() => a$.set('error in reactor')).to.throw('error in reactor');
+            });
+
+            it('should stop the reactor', () => {
+                a$.set('whatever');
+                expect(latestValue).to.equal('whatever');
+
+                try { a$.set('error in reactor'); } catch (e) {/**/ }
+                expect(latestValue).to.equal('error in reactor');
+
+                a$.set('back to normal?');
+                expect(latestValue).to.equal('error in reactor');
+            });
+
+            it('will break other reactors, and therefore your application', () => {
+                react(d$);
+                shouldHaveReactedOnce('no error');
+
+                try { a$.set('error in reactor'); } catch (e) {/**/ }
+                // use error handlers on unstable reactors to prevent this behavior
+                shouldNotHaveReacted();
+
+                a$.set('unstable reactors are bad');
+                shouldHaveReactedOnce('unstable reactors are bad');
+            });
+        });
+    });
+
+    context('with an error handler', () => {
+        context('when an error occurs in any derivation', () => {
+            let errorHandler: sinon.SinonSpy;
+            beforeEach('start the reactor', () => {
+                errorHandler = spy();
+                react(d$, { errorHandler });
+                shouldHaveReactedOnce('no error');
+                shouldNotHaveBeenCalled(errorHandler);
+            });
+
+            it('should not throw on Atom#set', () => {
+                expect(() => a$.set('error in derivation')).not.to.throw();
+            });
+
+            it('should call the errorhandler with the error', () => {
+                a$.set('error in derivation');
+                expect(errorHandler).to.have.been.calledOnce;
+                const err = errorHandler.firstCall.args[0];
+                expect(err).to.be.an('error');
+                expect(err.message).to.equal('error in derivation');
+
+                errorHandler.resetHistory();
+                a$.set('no error at all!');
+                expect(errorHandler).to.not.have.been.called;
+            });
+
+            it('should stop the reactor', () => {
+                a$.set('whatever');
+                shouldHaveReactedOnce('whatever');
+
+                a$.set('error in derivation');
+                shouldNotHaveReacted();
+
+                a$.set('no error at all!');
+                shouldNotHaveReacted();
+            });
```

Comment:
Maybe also spec what happens if an error is thrown directly after another error.